src/main/java/club/sk1er/hytilities/Hytilities.java

```diff
@@ -77,6 +78,7 @@ private void registerHandlers() {
         MinecraftForge.EVENT_BUS.register(new ChatHandler());
         MinecraftForge.EVENT_BUS.register(new AchievementEvent());
         MinecraftForge.EVENT_BUS.register(new LevelupEvent());
+        MinecraftForge.EVENT_BUS.register(new ConnectedMessage());
```

Comment:
this will not work, remove this line and use `this.registerModule(new ConnectedMessage())` in ChatHandler